lib/scheduler.pm

```diff
@@ -25,15 +25,17 @@ use File::Basename;
 use testapi qw(get_var set_var diag);
 use main_common 'loadtest';
 use YAML::PP;
+use YAML::PP::Schema::Include;
 use Data::Dumper;
 
 our @EXPORT = qw(load_yaml_schedule get_test_suite_data);
 
 my $test_suite_data;
-my $include_tag = '$include';
-
-my $ypp = YAML::PP->new();
+my $root_project_dir = dirname(__FILE__) . '/../';
 
+my $include = YAML::PP::Schema::Include->new(paths => ($root_project_dir));
+my $ypp     = YAML::PP->new(schema => [$include, 'Merge']);
```

Comment:
This might be not relevant yet because the scedule files only deal with strings right now, but usually you would want to add the default schema here:
```
YAML::PP->new(schema => ['+', $include, 'Merge']);
# or
YAML::PP->new(schema => ['Core', $include, 'Merge']);
```
The default schema is responsible for resolving numbers, undef and booleans.
I would recommend to enable it to have the default YAML 1.2 behaviour.

Reply:
Added. Thank you for the note.